### DISCUSSION TITLE
How do I remove the background highlighting from the linenumber column?

### DISCUSSION THREAD

**Discussion by rew1nter** [ellisonleao/gruvbox.nvim#344] [Q&A] [answered]
![image](https://github.com/ellisonleao/gruvbox.nvim/assets/64508791/5539aaad-173d-425f-ba60-75cee389e9a0)
I want to have the yellow foreground color of the line number but not the background highlight. Is it possible ?

**Comment by natemccurdy** [comment, marked as answer]
Set the `CursorLineNr` highlight group's background to empty.

For example, when using lazy.nvim, you'd set this in `config` for the plugin:
```lua
return {
  {
    "ellisonleao/gruvbox.nvim",
    config = function()
      require("gruvbox").setup({
        overrides = {
          CursorLineNr = { bg = "" },
        },
      })
    end,
  },
}
```
